src/System.Console/src/System.Console.builds

```diff
@@ -9,23 +9,8 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Console.csproj">
-      <OSGroup>Unix</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <TargetGroup>net46</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
       <TargetGroup>net463</TargetGroup>
     </Project>
-    <Project Include="System.Console.csproj">
```

Comment:
Please verify that this has actually shipped. I have a suspicious that we've never shipped this yet as we only recently added netcore50 support.